CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# CHANGELOG
+
+## v0.2.0
+
+### Summary
+
+This version introduces [http header matching](https://github.com/aws/aws-app-mesh-roadmap/issues/15) and [explicit priority to routes](https://github.com/aws/aws-app-mesh-roadmap/issues/77).
+
+### Changes
+
+* Update installation instruction for aws-app-mesh-inject (https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/60, @fmedery)
+* Fix unit tests and added Makefile target (https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/61, @kiranmeduri)
+* Using RetryOnConflict to deal with concurrent updates to resources (https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/64, @kiranmeduri)
+* Add http-header match and priority to routes (https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/56, @kiranmeduri)
```

Comment:
did retry-policy not make the cut?

Reply:
Same question for the virtual router name fix.

Reply:
I thought retry was there, but anyway I'll just add everything since my subjective picking and choosing what's important is a bad idea.

Reply:
No worries, it is still a massive improvement than what we have. Thanks for compiling the list.